engine/settings/celery_task_routes.py

```diff
@@ -33,15 +33,15 @@
     "apps.schedules.tasks.notify_about_gaps_in_schedule.check_empty_shifts_in_schedule": {"queue": "default"},
     "apps.schedules.tasks.notify_about_gaps_in_schedule.start_notify_about_gaps_in_schedule": {"queue": "default"},
     "apps.schedules.tasks.notify_about_gaps_in_schedule.check_gaps_in_schedule": {"queue": "default"},
-    "apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_empty_shifts_in_schedule": {"queue": "default"},
+    "apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_gaps_in_schedule_task": {"queue": "default"},
```

Comment:
`apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_empty_shifts_in_schedule` is a typo, it does not correspond to a real function.

I've simply moved `apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_gaps_in_schedule` up in the dict to be alongside the other `apps.schedules.tasks.notify_about_gaps_in_schedule` module tasks